singularity-opac/neutrinos/spiner_opac_neutrinos.hpp

```diff
@@ -61,7 +61,7 @@ enum class DataStatus { Deallocated, OnDevice, OnHost };
 template <typename ThermalDistribution>
 class SpinerOpacity {
  public:
-  static constexpr Real EPS = 10.0 * std::numeric_limits<Real>::epsilon();
+  static constexpr Real EPS = 10.0 * std::numeric_limits<Real>::min();
```

Comment:
should we remove the 10?

Reply:
I think the 10 is reasonable just to get a bit away from the "quantum foam" right at `::min()`, but I don't have great intuition here, if you think the 10 is bad I can remove it.